test/expected/chunk_utils.out

```diff
@@ -1601,3 +1601,58 @@ SELECT * FROM test_view_part_few WHERE ts BETWEEN '2024-01-04 00:00:00+00'AND '2
          15 | Thu Jan 04 21:55:12 2024 PST
 (14 rows)
 
+-- Hypertable by hash partitioning
+CREATE TABLE test_by_hash (id BIGINT, value FLOAT8);
+SELECT create_hypertable('test_by_hash', by_hash('id', 10));
+ create_hypertable 
```

Comment:
@fabriziomello with my PR https://github.com/timescale/timescaledb/pull/6993 we won't allow this.

Reply:
Fixed!